barrier3d/barrier3d.py

```diff
@@ -779,6 +756,413 @@ def migrate_dunes(
                 self._SCRagg[time_index] + cellular_shoreline_change
             )  # Reset (make more positive), leaving residual
 
+    @staticmethod
```

Comment:
can you add a comment here that says briefly what this does (basically the name of this pull request)

Reply:
Done, see new commit